wormhole-connect/src/routes/bridge/baseRoute.ts

```diff
@@ -25,9 +25,10 @@ import { hexlify } from 'ethers/lib/utils.js';
 
 export abstract class BaseRoute extends RouteAbstract {
   async isSupportedSourceToken(
-    token: TokenConfig | undefined,
-    destToken: TokenConfig | undefined,
+    token?: TokenConfig,
+    destToken?: TokenConfig,
     sourceChain?: ChainName | ChainId,
+    destChain?: ChainName | ChainId,
   ): Promise<boolean> {
     if (!token) return false;
```

Comment:
This function probably shouldn't even be called if the `token` is `undefined`, but I don't want to stray too far off into a refactoring adventure...